caluma/caluma_core/tests/test_visibility.py

```diff
@@ -167,6 +167,42 @@ class ConfiguredUnion(Union):
     assert queryset.count() == 0
 
 
+def test_union_visibility_return_managers(db, history_mock):
+    FakeModel = get_fake_model(
+        dict(name=CharField(max_length=255)), model_base=models.UUIDModel
```

Comment:
Any particular reason for `dict()` vs `{}`? Not really important, but I'd prefer the literal syntax

Reply:
It's a copy/paste from the test above.